api/overview/azure/dns.md

```diff
@@ -30,10 +33,30 @@ Install-Package Microsoft.Azure.Management.Dns.Fluent
 dotnet add package Microsoft.Azure.Management.Dns.Fluent
 ```
 
+### Example
+
+The following example creates a new DNS zone.
+
+```csharp
+/*
+using Microsoft.Azure.Management.Dns;
+using Microsoft.Azure.Management.Dns.Models;
+*/
+Zone dnsZoneParams = new Zone("global");
+dnsZoneParams.Tags = new Dictionary<string, string>();
+dnsZoneParams.Tags.Add("dept", "finance");
+Zone dnsZone =
+    await dnsClient.Zones.CreateOrUpdateAsync(resourceGroupName, zoneName, dnsZoneParams, null, "*");

```

Comment:
Might be helpful to include the code to create the client as well, as its constructor is a bit non-trivial.